scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -38,6 +41,11 @@
 # Latest sensor vs central versions in central_chart_versions
 test_tuples.extend([Chart_versions(central_version=central_chart_version, sensor_version=latest_tag) for central_chart_version in central_chart_versions])
 
+# Support exception for latest central and sensor 3.74 as per https://issues.redhat.com/browse/ROX-18223
+support_exceptions = [Chart_versions(central_version=latest_tag, sensor_version=get_latest_helm_chart_version_for_specific_release("stackrox-secured-cluster-services", Release(major=3, minor=74)))]
```

Comment:
Each item in the support exceptions could be in a new line to improve readability..

Reply:
Well so far there is only one support exception so no line breaks necessary yet